src/main/java/gregtech/common/metatileentities/storage/MetaTileEntityCreativeChest.java

```diff
@@ -188,12 +185,38 @@ public void addInformation(ItemStack stack, @Nullable World player, List<String>
     @Override
     public void receiveInitialSyncData(@NotNull PacketBuffer buf) {
         super.receiveInitialSyncData(buf);
-        this.handler.setStackInSlot(0, this.virtualItemStack);
+        this.creativeHandler.setStackInSlot(0, this.virtualItemStack);
     }
 
     @Override
     public IItemHandler getTypeValue() {
-        return this.handler;
+        return this.creativeHandler;
+    }
+
+    protected class ModifiableHandler extends QuantumChestItemHandler implements IItemHandlerModifiable {
+
+        @Override
+        public void setStackInSlot(int slot, ItemStack stack) {
+            virtualItemStack = GTUtility.copy(1, stack);
+            itemsStoredInside = stack.isEmpty() ? 0 : 1;
```

Comment:
I don't know if the creative chest would even care about these, since there can only be 1 item in the slot, and it is never storing anything virtualized.

Reply:
I have to update this since it's what is displayed in the item slot